src/module/api/service/event/event.service.ts

```diff
@@ -419,10 +419,34 @@ export class EventService {
       });
     });
 
-    if (prices?.length) {
+    if (aggregateOffer.additionalType[0].endsWith("Paid")) {
```

Comment:
Can the additional type be undefined or empty?

Reply:
Should we add it like this?
aggregateOffer.additionalType?.[0]?.toLowerCase()?.endsWith("paid")

Reply:
The additionalType URLs are of the format: http://kg.footlight.io/resource/Free and http://kg.footlight.io/resource/Paid. Do you want me to change the code to add toLoweCase?

Reply:
Okay, this looks like the URL will be constants. Please add this URL are constants or ENUM and do the comparison, please.
Also, make sure that the empty and undefined cases are also handled.

Reply:
I've updated the code. Please review it and let me know if you need any changes.